backend/slack_commands/interactive/form_handler.py

```
@@ -0,0 +1,295 @@
+CALLBACK_ID = "form_run_workflow"
+ACTION_SCOPE = "form_scope_select"
+ACTION_TEAM = "form_team_select"
+ACTION_WORKFLOW = "form_workflow_select"
+
+_SCOPE_PERSONAL = "personal"
+_SCOPE_TEAM = "team"
+
+
+class FormHandler:
+
+    def __init__(self, mas_url: str, identity_client: IdentityClient, executor: SessionExecutor):
```

Ruff (0.16.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)